src/main/java/com/salesforce/dynamodbv2/mt/mappers/sharedtable/impl/ConditionMapper.java

```diff
@@ -16,14 +16,21 @@
 interface ConditionMapper {
```

Comment:
I feel like some tests on these methods directly might help understand the code better.  Perhaps it's not necessary now and would be a further distraction.  I realize I should have given myself this advice when I did this in the first place.  :-|